test/common.js

```diff
@@ -27,6 +28,21 @@ common.httpsPort = 9443;
 common.httpsServerKey = fs.readFileSync(path.join(__dirname, './fixture/key.pem'));
 common.httpsServerCert = fs.readFileSync(path.join(__dirname, './fixture/cert.pem'));
 
+common.createServer = function (form, FIELDS, callback) {
+  return http.createServer(function (req, res) {
+
+    form.parse(req);
+
+    var fieldsPassed = Object.keys(FIELDS).length;
```

Comment:
Do you mind to put it to the top of the function?
To keep it with the spirit of hoisting. :)

And actually it would be good at the top of the `common.createServer`,
so it won't be redefined on each request, in case we'd get multi-request tests.